### ISSUE TITLE
Request for detailed hyperparameters

### ISSUE THREAD

**Issue by jeongwhanchoi** [yzhang1918/cikm2021cope#1]
Hi @yzhang1918 ,
I've read your work interestingly.
I want to reproduce your work, but there are no detailed hyperparameters for each dataset.

Could you provide detailed hyperparameters for each dataset?

Best,
Jeongwhan

**Comment by jeongwhanchoi** [comment]
I've tried to run with the default arguments because there are no reported hyperparameters.

I would like to ask about the loss and evaluation metrics.
I've found the training loss is decreasing while evaluation metrics are not increasing.
It seems that the reported results are from the first epoch, or the model is overfitted.

For that reason, could you provide detailed hyperparameters?

```
[Epoch: 0 ]
Loss=0.1365                                                                                                                     
------- Valid MRR: 0.0700 Recall@10: 0.1763 Recall@20: 0.2901
=======  Test MRR: 0.0742 Recall@10: 0.1822 Recall@20: 0.2901
[Epoch: 1 ]
Loss=0.1209                                                                                                                     
------- Valid MRR: 0.0703 Recall@10: 0.1741 Recall@20: 0.2698
=======  Test MRR: 0.0730 Recall@10: 0.1792 Recall@20: 0.2698
[Epoch: 2 ]
Loss=0.1054                                                                                                                     
------- Valid MRR: 0.0691 Recall@10: 0.1733 Recall@20: 0.2653
=======  Test MRR: 0.0697 Recall@10: 0.1664 Recall@20: 0.2653
[Epoch: 3 ]
Loss=0.0891                                                                                                                     
------- Valid MRR: 0.0603 Recall@10: 0.1598 Recall@20: 0.2381
=======  Test MRR: 0.0636 Recall@10: 0.1604 Recall@20: 0.2381
[Epoch: 4 ]
Loss=0.0754                                                                                                                     
------- Valid MRR: 0.0563 Recall@10: 0.1326 Recall@20: 0.2080
=======  Test MRR: 0.0559 Recall@10: 0.1461 Recall@20: 0.2080
[Epoch: 5 ]
Loss=0.0638                                                                                                                     
------- Valid MRR: 0.0484 Recall@10: 0.0972 Recall@20: 0.1530
=======  Test MRR: 0.0532 Recall@10: 0.1288 Recall@20: 0.1530
[Epoch: 6 ]
Loss=0.0551                                                                                                                     
------- Valid MRR: 0.0384 Recall@10: 0.0739 Recall@20: 0.1047
=======  Test MRR: 0.0389 Recall@10: 0.0851 Recall@20: 0.1047
[Epoch: 7 ]
Loss=0.0492                                                                                                                     
------- Valid MRR: 0.0351 Recall@10: 0.0625 Recall@20: 0.0950
=======  Test MRR: 0.0282 Recall@10: 0.0602 Recall@20: 0.0950
[Epoch: 8 ]
Loss=0.0433                                                                                                                     
------- Valid MRR: 0.0424 Recall@10: 0.0746 Recall@20: 0.1063
=======  Test MRR: 0.0317 Recall@10: 0.0678 Recall@20: 0.1063
[Epoch: 9 ]
Loss=0.0379                                                                                                                     
------- Valid MRR: 0.0374 Recall@10: 0.0746 Recall@20: 0.1010
=======  Test MRR: 0.0339 Recall@10: 0.0776 Recall@20: 0.1010
[Epoch: 10 ]
Loss=0.0349                                                                                                                     
------- Valid MRR: 0.0410 Recall@10: 0.0731 Recall@20: 0.1115
=======  Test MRR: 0.0340 Recall@10: 0.0745 Recall@20: 0.1115
```

**Comment by Zhangjunheeee** [comment]
Same question here.Could you help let us know how to solve this problem？
